Sources/MongoSwift/SDAM.swift

```diff
@@ -140,7 +140,7 @@ public struct ServerDescription {
     /// mongoc_server_description_t.
     internal init(_ description: OpaquePointer) {
         self.address = Address(mongoc_server_description_host(description))
-        self.roundTripTime = mongoc_server_description_round_trip_time(description)
+        self.roundTripTime = Int(mongoc_server_description_round_trip_time(description))
```

Comment:
similar to the command duration one this seems ok too... this one is even less likely to be an issue because it's milliseconds rather than microseconds